docs/reference/eql/functions.asciidoc

```diff
@@ -482,6 +483,65 @@ If using a field as the argument, this parameter does not support the
 *Returns:* string or `null`
 ====
 
+[discrete]
+[[eql-fn-stringcontains]]
+=== `stringContains`
+
+Returns `true` if a source string contains a provided substring.
+
+[%collapsible]
+====
+*Example*
+[source,eql]
+----
+// process.command_line = "start regsvr32.exe"
+stringContains(process.command_line, "regsvr32")  // returns true
+stringContains(process.command_line, "start ")    // returns true
+stringContains(process.command_line, "explorer")  // returns false
```

Comment:
I would add one additional example, to show a less obvious usage: `stringContains(process.path, process.name)`.

Reply:
Added with 5ccc1986852. Thanks!